game/gui/hud.py

```diff
@@ -10,12 +10,15 @@
 
 HP_WIDTH = 150
 HP_HEIGHT = 20
+SCREEN_WIDTH = 1000
+SCREEN_HEIGHT = 650
 HP_BORDER = 2
 HP_X = 10 + HP_WIDTH / 2 + HP_BORDER
 HP_Y = 10 + HP_HEIGHT / 2 + HP_BORDER
 HP_BORDER_COLOR = (255, 255, 255)
 HP_BAR_COLOR = (0, 255, 0)
 HP_TEXT_MARGIN = 5
+EFFECT_TIME = 0.3
```

Comment:
A better name would be `DAMAGE_FLASH_SECS`. It's more specific in saying what "effect" you're referring to (what if we had more than one effect?) and it specifies the unit of time that this constant is in.